pkgs/tools/games/alice-tools/default.nix

```diff
@@ -0,0 +1,109 @@
+{ stdenv
+, lib
+, gitUpdater
+, fetchFromGitHub
+, fetchpatch
+, meson
+, ninja
+, pkg-config
+, bison
+, flex
+, libiconv
+, libpng
+, libjpeg
+, libwebp
+, zlib
+, withGUI ? true
+, qtbase ? null
+, wrapQtAppsHook ? null
+}:
+
+assert withGUI -> qtbase != null && wrapQtAppsHook != null;
+
+stdenv.mkDerivation rec {
+  pname = "alice-tools";
+  version = "0.12.1";
+
+  src = fetchFromGitHub {
+    owner = "nunuhara";
+    repo = "alice-tools";
+    rev = version;
+    fetchSubmodules = true;
+    hash = "sha256-uXiNNneAOTDupgc+ZvaeRNbEQFJBv4ppdEc3kZeUsg8=";
+  };
+
+  patches = [
+    # Improves DCF & PCF parsing
+    # Remove when version > 0.12.1
```

Comment:
Remove one or both of these? We probably should clarify if this comment extends to both patches

Reply:
One patch is for the `libsys4` submodule which handles the format parsing & converting, the other for `alice-tools` which provides the CLI/GUI and calls into the library. Both need a patch for it to work. Will clarify with a comment.

Reply:
Edited the comment so it refers to both patches.